packages/perseus/src/validation.types.ts

```diff
@@ -124,7 +123,9 @@ export type PerseusInteractiveGraphRubric = {
 
 export type PerseusInteractiveGraphUserInput = PerseusGraphType;
 
-export type PerseusLabelImageRubric = PerseusLabelImageWidgetOptions;
+/* TODO(LEMS-2440): Should be removed or refactored. Grading info may need
+    to be moved to the rubric from userInput. */
+export type PerseusLabelImageRubric = Empty;
```

Comment:
I originally just removed everything related to rubric, but I think we might want to pull `answers` out of `userInput`.